lessons/cryptography/learn-to-count/learn-to-count.md

```diff
@@ -0,0 +1,105 @@
+# [Cryptography] Learn to Count
+*Learn about why there are different number systems.*
+
+![banner alt text](.rsrc/banner.png)
+
+# Objectives
+- Learn the following number systems: decimal, binary, hexadecimal
+
+# Introduction
+When we think of numbers, we are usually thinking about the decimal number system. This system that we are used to uses 10 as a base, and the digits range from 0 to 9 (total of 10 digits). There are also other number systems, such as binary and hexadecimal. The binary number system uses 2 as a base, and the digits range from 0 to 1 (total of 2 digits). For example, the decimal number 42 would be 0b101010 in binary. The hexadecimal number system uses 16 as a base, and the digits range from 0 to 9, and then A-F (total of 16 digits). For example, the decimal number 42 would be 0x2A in hexadicimal. 
```

Comment:
we need to describe this format "0b101010" or they are going to be confused about the 0b, same deal with the 0x in hex

Reply:
Updated